### ISSUE TITLE
BUG: Query Parameter no longer working

### ISSUE THREAD

**Issue by uhu-ditoy** [beekeeper-studio/beekeeper-studio#1353]
**Describe the bug**
Query parameters like `:param` or `$1` are not working any more. Instead a MySQL/MariaDB Error is displayed

**To Reproduce**
1. Go to SQL editor in Beekeper Studio
2.enter a query like `select * from table where foo = :one and bar = :two`
3. See error: `You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'table where foo = :one and bar = :two' at line 1`

**Expected behavior**
Beekeper Studio should als for the parameters, as shown in the documentation

**Version Information (please complete the following information):**
 - OS: MacOS 12.5 (Monterey)
 - App Version: 3.6.2
 - Database type and version_ MariaDB 10.6.3

**Comment by swtseaman** [comment]
I have the same problem. I don't know what to do

**Comment by not-night-but** [comment]
Yeah I've noticed a couple weird things with the parameter system. Currently we don't manage parameters ourselves, I believe they are managed by the query builder we use, and only certain types of params are supported depending on the database.

Personally I think we should build our own parameter replacement system so we can offer the same functionality to every database no matter what knex supports.

**Comment by rathboma** [comment]
Duplicate of #1100.
